pyoptsparse/pyOpt_optimization.py

```diff
@@ -1642,7 +1642,8 @@ def __str__(self):
                 else:
                     raise ValueError(f"Unrecognized type for variable {var.name}: {var.type}")
 
-                text += fmt.format(idx, var.name, var.type, lower, value, upper, status, width=num_c)
+                if not (minimal_print and not status):
```

Comment:
A more clear logic would be `if status or not minimal_print`

Reply:
If status and mininal_print were both false, that would evaluate as true.